resolwe/flow/models.py

```diff
@@ -105,6 +105,13 @@ class Meta(BaseModel.Meta):
 
     data = models.ManyToManyField('Data')
 
+    #: collection descriptor schema
+    descriptor_schema = models.ForeignKey('DescriptorSchema', blank=True, null=True, on_delete=models.PROTECT)
```

Comment:
Yes. Please check all models.PROTECT references and think if such fix is appropriate.